zeppelin-web/pom.xml

```diff
@@ -120,34 +152,77 @@
           </execution>
 
           <execution>
-            <id>yarn install</id>
-            <goals>
-              <goal>yarn</goal>
-            </goals>
-            <configuration>
-              <arguments>install --no-lockfile</arguments>
-            </configuration>
+              <id>yarn add bower</id>
+              <goals>
+                  <goal>yarn</goal>
+              </goals>
+              <configuration>
+                <!--ignore-scripts-->
+                  <arguments>add bower --no-lockfile --no-bin-links</arguments>
```

Comment:
we don't need to run

'yarn add bower'
'yarn add karma-phantomjs-launcher'

and we can just run command defined in 'script' field in 'package.json'.

like 

'yarn install'
'yarn build'
'yarn test'

we don't want to define the same command in the pom.xml that 'package.json' already has in 'script' field, such as

'bower install'
'webpack build'
'grunt pre-webpack-dist'
'grunt post-webpack-dist'
'karma test'


I think all changes made in 'executions' section of 'frontend-maven-plugin' are unnecessary and should be reverted.